packages/com.pirate.refresh.yml

```diff
@@ -0,0 +1,20 @@
+title: Auto Dev Token Refresh
+iconUri: https://github.com/LucifersCircle/WebOS-Token-Refresh-App/blob/main/src/app/icon.png?raw=true
+manifestUrl: https://raw.githubusercontent.com/LucifersCircle/WebOS-Token-Refresh-App/refs/heads/main/manifest.json
```

Comment:
The same thing applies to the manifest, but since it changes through versions, we could use this.

I would suggest using releases instead, please let me know if you need help creating that!